src/test/javascript/spec/component/organization/organization-management-detail.component.spec.ts

```diff
@@ -140,12 +144,110 @@ describe('OrganizationManagementDetailComponent', () => {
                 { id: user1.id, login: user1.login },
                 { id: user2.id, login: user2.login },
             ]);
-            expect(component.searchNoResults).toEqual(false);
-            expect(component.searchFailed).toEqual(false);
+            expect(component.searchNoResults).toBe(false);
+            expect(component.searchFailed).toBe(false);
+        });
+
+        tick(100);
```

Comment:
Would it be possible to achieve the same effect with `tick()`? If the aim is advancing micro/macro task queue only and no milliseconds need to pass, leaving out `100` might make the intent more clear.

Same goes for other `tick(100)` usages below.